### ISSUE TITLE
LAB 19

### ISSUE THREAD

**Issue by UsingCoding** [ondrop/basics-of-programming#3]
### 19.1

- [x] Файл вам здесь определенно не нужен

- [x] Ваш алгоритм перебора можно упростить: вам стоит не искать число кратное прибаляя 1 и деля, искать кратные можно по другому. Умножать простое число последовательно, пока не дойдёте до макс значения

- [x] Так же стоит это упростить
https://github.com/ondrop/basics-of-programming/blob/53be8ee9dbd13a0bbb42567094c0cf8edadff2ba/Lw19/191.pas#L16-L37

**Comment by ondrop** [comment]
Исправил

**Comment by UsingCoding** [comment]
### 19.1

- [x] Есть ошибки форматирования

**Comment by UsingCoding** [comment]
### 19.2 

- [x] Почему-то при предоставленном вами инпуте программа выводит `JAN  1`  и всё

- [x] Неиспользуемые переменные
https://github.com/ondrop/basics-of-programming/blob/041c2aec587995fb3be8779ef998fe60def52ac7/Lw19/SortDateModule.pas#L52

**Comment by ondrop** [comment]
19.1 исправил.
Дело было в круглых скобках?Как я понял, перед скобками не нужны пробелы. Если не в них дело, то я не вижу ошибок

**Comment by UsingCoding** [comment]
> 19.1 исправил.
> Дело было в круглых скобках?Как я понял, перед скобками не нужны пробелы. Если не в них дело, то я не вижу ошибок

Ошибки по-прежнему остались

**Comment by ondrop** [comment]
Я дал проверить нескольким людям, но ошибок так и не нашлось. Можете показать где она? Готов отдать за это балл, если это и правда ошибка . Может это ошибка, о которой я ещё не знаю. Перепроверил по кодинг конвеншону. Ошибок не нашёл

**Comment by ondrop** [comment]
Исправил 19.2

**Comment by UsingCoding** [comment]
> Я дал проверить нескольким людям, но ошибок так и не нашлось. Можете показать где она? Готов отдать за это балл, если это и правда ошибка . Может это ошибка, о которой я ещё не знаю. Перепроверил по кодинг конвеншону. Ошибок не нашёл

Я не подкупен и объективен )

**Comment by ondrop** [comment]
Тогда можно хотя бы узнать в чем ошибка?

**Comment by ondrop** [comment]
И можно узнать коэффициенты были коэффициенты для лабораторных 17-19. Иначе баллы нужно ставить, а коэффициенты неизвестны.

**Comment by UsingCoding** [comment]
> Тогда можно хотя бы узнать в чем ошибка?

Конечно, мы же пользуемся машиной времени - вы можете перейти на коммит с подписью `Исправлены ошибки форматирования` и на 26 строке увидите ту самую ошибку

**Comment by UsingCoding** [comment]
> И можно узнать коэффициенты были коэффициенты для лабораторных 17-19. Иначе баллы нужно ставить, а коэффициенты неизвестны.

К сожалению, я сам ещё не знаю

**Comment by ondrop** [comment]
Нам сказали, что решение по коэффициентам принимают преподаватели, которые выставляют оценки.

**Comment by ondrop** [comment]
На 26 строке стоит обычное присваивание. После него не нужно точки с запятой и не нужно перед ним два пробела, так как он не входит в цикл на 23 строке. Так что же в нем не так?
